test/acceptance/typography.test.js

```diff
@@ -28,8 +28,8 @@ var browsers = {
 	})
 };
 
-var endpoint = 'http://localhost:8000/components/d2l-typography/test/acceptance/typography.html';
-var endpointMixins = 'http://localhost:8000/components/d2l-typography/test/acceptance/typography-mixins.html';
+var endpoint = 'http://localhost:8081/components/d2l-typography/test/acceptance/typography.html';
```

Comment:
@svanherk : another thing to watch out for.

Reply:
Yeah all the galen stuff seems different, so I was thinking just saving patches for these (@awikkerink already had one for this).  I'll add it to the converter repo.

Reply:
I'm not sure how rerunable this will be.  As soon as some legit change happens in the p3/master version, I don't think we'll be able to rerun the script.

Reply:
The script or the patch?  I'm not too worried about the patches, especially in galen, changes don't happen often there.  The change from ``` ` ``` to `'` will be interesting, I'm not sure how smart git is with its patches